weekly_meeting_notes/sept_20.md

```diff
@@ -0,0 +1,84 @@
+# 🚀 Sept 20th 
+**Start Time** 9:30am
+
+**End Time** 10:30am
+
+**Location**
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Edouard 
+2. Veronica
+3. Logan 
+4. Matt
+
+***Attendence Comments***
+
+
+## 🧸 Check-In
+
+- Dad joke of the day: "Our team had a debate for best names for looping variables ? "
+  - i won
+
+
+## Announcements
+
+- Client meeting on Friday @ 4:00pm, Location TBD
+- Team Exercise Due Friday
+- Team agreement doc Due (Submitted already!)
+
+## Last Week (Retropestive)
+
+- Verionica worked on the team agreement
+  - Logan and Edouard commented and approved pull request
+
+- Logan set up repo and made it so that 2 reviewers are required to approve PR's 
+
+## This Week  
+
+### Client meeting 
+
+- Matt will rebook the Client meeting location
+  - Two buisness days for booking request
+- Agenda needs to be made with the other Client Liasons
+  - Have questions sent to matt by Thursday Sept 22nd
+- Meeting space is TBD
+  - Matt will look at back-up options:
+    - Email the professor
+
+### Team Exercise
+
+- Team exerise is due on friday
+  - Submission should made in another repoGitHub repo
+
+- Each feature should have a test case
+- Programming idea's:
+    1) Pre processing a text file idea: 
+        - Reading from a text and produce string (Edouard)
+        - Pull out punctionation, lowercase, and word count (Verionca)
+        - Sort words alphabetically (Logan)
+        - Sort words randomly (Matt)
+  - TEST STRING = "Our team had a debate for best names for looping variables ? i won"
+- Everyone should write their own github issue
+
+    1) Sorting algorithm idea
+        - Each team does a sorting algorithm
+
+- Logan Set up repo
+- Group selected programming exercise idea 2
+
+
+## Board Update
+
+- Exploration: Finding a string for test case
+- Task x 4 (Each feature listed above)
+
+## To Do List
```

Comment:
I appreciate having a to-do list, thanks for adding this section!

Reply:
no problem!